doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -133,6 +133,14 @@ on ``mon.a``).
    For example::
 
 	ceph-mon -i {mon-id} --public-addr {ip:port}
+	
+
+#. add the entry in the ceph.conf configuration file.
```

Comment:
might want to note that 

1. this step is *optional*. as the clients are able to discover and fallback to the new monitor if it is able to connect to the cluster. but it helps to share the load from a newly started a client.
2. please replace, for instance, `d` in `[mon.d]` with `[mon{mon-id}]` or note that these settings are subject to change according to the actual settings of the newly added monitor.

Reply:
and the title could be improved, as the added step belongs to the section of "Adding a Monitor (Manual)".

Reply:
```
_Optional -- only complete this step if {insert reason}:_ Add the entry in the ceph.conf configuration file. For example:
```